adafruit_us100.py

```diff
@@ -25,15 +25,21 @@
 
 import time
 
+try:
+    from typing import Union
+    from busio import UART
+except ImportError:
+    pass
+
 
 class US100:
     """Control a US-100 ultrasonic range sensor."""
 
-    def __init__(self, uart):
+    def __init__(self, uart: UART):
         self._uart = uart
 
     @property
-    def distance(self):
+    def distance(self) -> float:
```

Comment:
Since this can return `None`, it should be `Optional[float]`.  Can you correct the docstring too? Thanks!